src/deva/utils/ssh.py

```diff
@@ -0,0 +1,66 @@
+# SPDX-FileCopyrightText: 2024-present Datadog, Inc. <dev@datadoghq.com>
+#
+# SPDX-License-Identifier: MIT
+from __future__ import annotations
+
+from deva.utils.fs import Path
+
+RELATIVE_CONFIG_DIR = ".deva"
+
+
+def ssh_config_dir() -> Path:
+    return Path.home() / ".ssh"
+
+
+def ssh_base_command(destination: str, ssh_port: int | str) -> list[str]:
+    return [
+        "ssh",
+        # Enable agent forwarding
+        "-A",
+        # Silence redundant output
+        "-q",
+        # Allocate pseudo-terminal, required for colored output and some commands
+        "-t",
+        "-p",
+        str(ssh_port),
+        destination,
+        # Indicate that there are no more flags
+        "--",
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**
These comments are great!